src/main/resources/public/raml/common-schema.raml

```diff
@@ -130,6 +136,12 @@ subscriptionList: |
           },
           "userName":{
              "type":"string"
+          },
```

Comment:
remove restUser and token. Front end will have to add it to formValue in notificationMessageKeyValues

Reply:
Done